app/views/layouts/shared/_google_analytics.html.slim

```diff
@@ -0,0 +1,8 @@
+- if Rails.env == "production"
+  /! Global site tag (gtag.js) - Google Analytics
+  script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-112609815-1"
```

Comment:
move ID to secrets

Reply:
actually don't move to secrets.
this is not a private info.

BUT
do not convert it to slim

use ERB with same code as you got from google